### DISCUSSION TITLE
Adding new sensor classes - number

### DISCUSSION THREAD

**Discussion by raccettura** [home-assistant/architecture#614] [Entity Models] [answered]
Recently ran across this scenario and don't see a way to solve it currently.  I integrated my ads-b antenna into home assistant and am tracking some stats like # of flights and # of messages received.  I want those numbers to be graphed, but seems like device class "none" is treated more like a string where HA's frontend for example just tracks the change, not the value making useless (but pretty) charts like this:
![image](https://user-images.githubusercontent.com/126830/130363321-e20e6c59-f7d7-4497-a8c1-81de68085b3c.png)

While this use is too specific for adding specific classes, I'd propose adding a numeric class that's treated like a float and perhaps graphed like any other temperature, power, pressure, humidity.  Then "none" is essentially a string state.

**Comment by frenck** [comment, marked as answer]
Currently, we rely on `unit_of_measurement`, if one is defined, it is a numerical value. However, we are currently in the process of rolling out `state_class`. Right now, mainly used for the energy stuff and the new statistics.

In the future, it will also solve this issue (which is mainly a tasks that needs to be done, not a limitation).

**Reply by andriej** [reply]
@raccettura I had same issue and just added space as unit of measurement. 
Let's call it a hack, but I agree it will be nice to have it as sensor in future - just as Frenck said.

**Reply by raccettura** [reply]
@andriej 
![image](https://user-images.githubusercontent.com/126830/130365566-79f905a7-e7bd-4224-82be-12b813028fca.png)

Not perfect, but way closer to what was intended and way more useful.

**Comment by fsaris** [comment]
I think this is resolved in the meantime. Adding a `state_class` to your sensor removes the need of the space hack.
